packages/apps/composer-app/vite.config.ts

```diff
@@ -72,7 +71,6 @@ export default defineConfig({
         resolve(__dirname, './index.html'),
         resolve(__dirname, './src/**/*.{js,ts,jsx,tsx}'),
         resolve(__dirname, './node_modules/@dxos/react-ui/dist/**/*.mjs'),
-        resolve(__dirname, './node_modules/@dxos/react-uikit/dist/**/*.mjs'),
```

Comment:
I think react-components will probably be needed here as well

Reply:
`react-ui` will be replaced by `react-components`.